[2025-01-13T18:01:54.191+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_date.md_account_d manual__2025-01-13T18:01:42.558465+00:00 [queued]>
[2025-01-13T18:01:54.209+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_date.md_account_d manual__2025-01-13T18:01:42.558465+00:00 [queued]>
[2025-01-13T18:01:54.210+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-13T18:01:54.515+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): md_account_d> on 2025-01-13 18:01:42.558465+00:00
[2025-01-13T18:01:54.536+0000] {standard_task_runner.py:60} INFO - Started process 2312 to run task
[2025-01-13T18:01:54.555+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_date', 'md_account_d', 'manual__2025-01-13T18:01:42.558465+00:00', '--job-id', '3047', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpu3227e__']
[2025-01-13T18:01:54.562+0000] {standard_task_runner.py:88} INFO - Job 3047: Subtask md_account_d
[2025-01-13T18:01:54.701+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_date.md_account_d manual__2025-01-13T18:01:42.558465+00:00 [running]> on host 711845a6becd
[2025-01-13T18:01:54.915+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='budanovsa' AIRFLOW_CTX_DAG_ID='insert_date' AIRFLOW_CTX_TASK_ID='md_account_d' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T18:01:42.558465+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T18:01:42.558465+00:00'
[2025-01-13T18:01:55.199+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-13T18:01:55.226+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_date, task_id=md_account_d, execution_date=20250113T180142, start_date=20250113T180154, end_date=20250113T180155
[2025-01-13T18:01:55.240+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-13T18:01:55.347+0000] {sql.py:450} INFO - Running statement: 
        INSERT INTO log.logt (execution_datetime, event_datetime, event_name)
        VALUES ('2025-01-13 18:01:42.558465', '2025-01-13T18:01:55.229615', 'md_account_d');
    , parameters: None
[2025-01-13T18:01:55.353+0000] {sql.py:459} INFO - Rows affected: 1
[2025-01-13T18:01:55.408+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-13T18:01:55.564+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
